scripts/tasks/babel.ts

```diff
@@ -0,0 +1,45 @@
+import { transformAsync } from '@babel/core';
+import * as glob from 'glob';
+import fs from 'fs';
+import path from 'path';
+import { EOL } from 'os';
+
+const OEL_REGEX = new RegExp(EOL, 'g');
+
+function addSourceMappingUrl(code: string, loc: string): string {
+  return code + '\n//# sourceMappingURL=' + path.basename(loc);
```

Comment:
Won't the code already have a `sourceMappingURL` in some cases (if the TS was built with sourcemaps)?

Reply:
Babel keeps stripping this comment, even when correct option is set. It's by design that we should restore it 🙂
I added a comment to avoid confusion